tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js

```diff
@@ -159,5 +192,29 @@ context('OpenCV. Intelligent scissors. Histogram Equalization.', () => {
             });
             cy.get('body').trigger('keydown', { keyCode: keyCodeN, code: 'KeyN' }).trigger('keyup');
         });
+
+        it('Create a shape with "TrackerMIL". Track it for several frames.', () => {
+            // Track shape and move from 0 to 1 frame to init tracker
+            // We will start testing tracking from 2 frame because it's a bit unstable on inintialization
+            cy.opencvTrackObject(createOpencvTrackerShape);
+            cy.goToNextFrame(1);
+            cy.get('#cvat_canvas_shape_3')
+                .invoke('attr', 'x')
+                .then((defaultX) => {
+                    cy.get('#cvat_canvas_shape_3')
+                        .invoke('attr', 'y')
+                        .then((defaultY) => {
+                            for (let i = 2; i < imagesCount; i++) {
+                                cy.goToNextFrame(i);
+                                cy.get('#cvat_canvas_shape_3').invoke('attr', 'x').then((xVal) => {
+                                    expect(parseFloat(xVal)).to.be.closeTo(Math.round(defaultX) + (i - 1) * 5, 2.5);
```

Comment:
Could you please clarify this expression: Math.round(defaultX) + (i - 1) * 5 ??

If tracker does not work, it also will be close to default value, maybe to check also that object has a keyframe?

Reply:
In the beginning of the test we created images with text, on each next image text is moved by 5px so in expression `x + (i-1)*5` we calculate expected coords assuming start from second frame, then we check if new coords are close to expected. I've reduced precision to 1.0 and added a keyframe check.
I think if tracking fails it wont be close enough to next expected value, most likely it will stay on the same place.